### DISCUSSION TITLE
Quantification output

### DISCUSSION THREAD

**Discussion by mavershang** [COMBINE-lab/salmon#627] [Q&A (bulk specific)]
Hi there, a quick question: suppose I use paired fq files, in the quan.sf, the NumOfReads represents the "number of reads mapped" or "the number of pairs mapped"?

Thanks!

**Comment by rob-p** [comment]
Hi @mavershang,

  The "NumReads" column always records the number of fragments.  So in a paired-end dataset, this will be read pairs (or orphaned reads if there is no proper alignment of the pair as a whole).

**Reply by mavershang** [reply]
Thank you @rob-p . A following question: in some cases, I built lib using many short sequences (<100bp), which is shorter than the fragment insert size. Will that affect the interpretation of the reads of fragment?

**Reply by alannaspiteri** [reply]
Hi @rob-p and @mavershang, 

Quick question. Are unmapped reads written in the 'gene quantification' file? i.e. does a value of '0' for the 'num of reads' and 'TPM' mean that 0 reads were mapped to the transcript? If so is there a way to only include mapped reads in that file? 

Thanks very much!
